test/create_env_cluster_topics_test.go

```diff
@@ -76,30 +57,189 @@ func TestEnvClusterTopic(t *testing.T) {
+	log.Println("Validating Kafka Topic Names...")
+	topicOutput := runCLICommand(t, "confluent", "kafka", "topic", "list", "-o", "json")
+	var topics []map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(topicOutput), &topics), "Failed to parse Kafka Topics JSON output")
+	kafkaTopicNames := []string{kafkaTopicName1, kafkaTopicName2}
+
+	// Validate each topic exists in the CLI output
+	for _, expectedTopicName := range kafkaTopicNames {
+		found := false
+		for _, topic := range topics {
+			if topic["name"] == expectedTopicName {
+				found = true
+				break
+			}
+		}
+		a.True(found, fmt.Sprintf("Expected topic '%s' not found in Kafka Topics list", expectedTopicName))
+		log.Printf("Validated Kafka Topic '%s' successfully.", expectedTopicName)
+	}
 
-		output = terraform.Output(t, runOptions, "kafka_topic_name")
-		a.Equal(output, "squad_raw_service_example_1_entity")
+	// Validate Sink Connector IDs
+	log.Println("Validating Sink Connector IDs...")
+	connectClustersOutput := runCLICommand(t, "confluent", "connect", "cluster", "list", "-o", "json")
+	var connectClusters []map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(connectClustersOutput), &connectClusters), "Failed to parse Connect Clusters JSON output")
+
+	gcsSinkFound := false
+	bigQuerySinkFound := false
+
+	for _, cluster := range connectClusters {
+		if cluster["id"] == gcsSinkConnectorID {
+			gcsSinkFound = true
+		}
+		if cluster["id"] == bigQueryConnectorID {
+			bigQuerySinkFound = true
+		}
+		if gcsSinkFound && bigQuerySinkFound {
+			break
+		}
+	}
 
-		output = terraform.Output(t, runOptions, "bigquery_connector_id")
-		a.NotEmpty(output)
+	a.True(gcsSinkFound, fmt.Sprintf("GCS Sink Connector ID '%s' not found in Connect Clusters list", gcsSinkConnectorID))
+	a.True(bigQuerySinkFound, fmt.Sprintf("BigQuery Connector ID '%s' not found in Connect Clusters list", bigQueryConnectorID))
+	log.Printf("Validated Sink Connector IDs '%s' and '%s' successfully.", gcsSinkConnectorID, bigQueryConnectorID)
+
+	// Validate Topic Service Account ID
+	log.Println("Validating Topic Service Account ID...")
+	serviceAccountsOutput := runCLICommand(t, "confluent", "iam", "service-account", "list", "-o", "json")
+	var serviceAccounts []map[string]interface{}
+	a.NoError(json.Unmarshal([]byte(serviceAccountsOutput), &serviceAccounts), "Failed to parse Service Accounts JSON output")
+
+	serviceAccountFound := false
+	for _, serviceAccount := range serviceAccounts {
+		if serviceAccount["id"] == topicServiceAccountID {
+			serviceAccountFound = true
+			break
+		}
+	}
+	assert.True(t, serviceAccountFound, fmt.Sprintf("Topic Service Account ID '%s' not found in service account list", topicServiceAccountID))
+	log.Printf("Validated Topic Service Account ID '%s' successfully.", topicServiceAccountID)
+}
 
-		output = terraform.Output(t, runOptions, "connector_gcs_sink_connector_id")
-		a.NotEmpty(output)
-	})
+// Run a generic Confluent CLI command and return output
```

Comment:
```suggestion
// Run a generic CLI command and return output
```

This function has nothing to do with Confluent it seems?

Reply:
fixed